.github/workflows/_lint.yaml

```diff
@@ -0,0 +1,62 @@
+name: Lint package
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        required: true
+        type: string
+
+      numpy:
+        required: true
+        type: string
+
+      python:
+        required: true
+        type: string
+
+      activate_command:
+        required: true
+        type: string
+
+jobs:
+  lint:
+    runs-on: ${{ inputs.os }}
+    name: lint-doc-${{ inputs.os }}-np${{ inputs.numpy }}-py${{ inputs.python }}
+    env:
+      TERM: xterm-256color  # To colorize output of make tasks.
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python }}
+
+      - name: Use zstd for faster cache restore (windows)
```

Comment:
I don't see how setting GITHUB_PATH make the cache use zstd ?

Reply:
This a recipe that comes straight from the doc. When we tested in actually worked.

Reply:
Which doc are you referring to ? I don't find anything on the relationship between GITHUB_PATH and zstd.

Reply:
I meant the Github Doc, however I do not find that piece of code anymore https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows . I suspect it is because of this particular version of windoes as with new python versions they use a new one, as with ubuntu.